docs/awesome_openxla.md

```diff
@@ -0,0 +1,59 @@
+# Awesome OpenXLA
```

Comment:
Can you also add a link in `_toc.yaml`?

Reply:
sg, adding under "title: Getting started"